.github/workflows/build.yml

```diff
@@ -1,5 +1,5 @@
 name: Build
-on: [pull_request]
+on: [push]
```

Comment:
Why the change?

Reply:
I want this workflow to run against any push (both PRs and pushes to master)